x-pack/plugins/siem/public/containers/timeline/details/index.tsx

```diff
@@ -32,7 +31,7 @@ export class TimelineDetailsComponentQuery extends React.PureComponent<TimelineD
     return executeQuery ? (
       <Query<GetTimelineDetailsQuery.Query, GetTimelineDetailsQuery.Variables>
         query={timelineDetailsQuery}
-        fetchPolicy={getDefaultFetchPolicy()}
+        fetchPolicy="network-only"
```

Comment:
I assume this and the update below are what resolve the [two issues](https://github.com/elastic/kibana/pull/33939#discussion_r269815362) @FrankHassanabad was references the last time the default fetch policy changed?

Reply:
Yes, I am trying to make @FrankHassanabad happy!!! That's my only reward!!! LOL

Reply:
haha, that is pretty funny